modules/scrabble.js

```diff
@@ -55,5 +55,19 @@ module.exports = {
         return String(scrabbleScore(msg.args.word) || 'Not a valid word')
       }
     }
+  },
+  events: {
+    message: function (bot, nick, to, text) {
+      var wordScores = {}
+      for (const word of text.split(' ')) {
+        wordScores[word] = scrabbleScore(word)
+      }
+      const bestWord = Object.keys(wordScores)
+        .reduce((a, b) => wordScores[a] > wordScores[b] ? a : b)
+      if (wordScores[bestWord] > 20 && !text.match('!scrabble')) {
+        bot.shout(to, nick + ': ' + bestWord + ' scores ' +
```

Comment:
you could use a template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here